extensions-rhel-9.6.yaml

```diff
@@ -0,0 +1,90 @@
+# RPMs as operating system extensions, distinct from the base ostree commit/image
+# https://github.com/openshift/enhancements/blob/master/enhancements/rhcos/extensions.md
+# and https://github.com/coreos/fedora-coreos-tracker/issues/401
+
+extensions:
+  # https://issues.redhat.com/browse/RFE-4177
+  # Not available in c9s
+  wasm:
+    architectures:
+      - x86_64
+      - aarch64
+    repos:
+      - rhel-9.4-server-ose-4.18
+      - rhel-9.4-appstream
```

Comment:
Why did we add `rhel-9.4-appstream` here when compared to the same section in `extensions-rhel-9.4.yaml` ?